packages/legend-query/src/models/protocols/pure/v1/V1_QueryBuilder_FunctionExpressionBuilder.ts

```diff
@@ -47,8 +47,64 @@ import {
   AbstractPropertyExpression,
   TYPICAL_MULTIPLICITY_TYPE,
   VariableExpression,
+  PRIMITIVE_TYPE,
 } from '@finos/legend-graph';
 
+const generateGenericTypeForBaseSimpleFunctionExpression = (
```

Comment:
I think this is the wrong place to do this, let's leverage this to `V1_getExtraFunctionExpressionBuilders` level